urlchecker/core/urlproc.py

```diff
@@ -101,10 +104,10 @@ def __init__(
         file_name=None,
         white_listed_patterns=None,
         white_listed_urls=None,
-        print_all=True,
+        print_level=None,
     ):
         self.file_name = file_name
-        self.print_all = print_all
+        self.print_level = print_level
```

Comment:
Especially if print_level is default to None here, setting the variable again needs to be:

```python
self.print_level = print_level or "all"
```